src/website/app/demos/Avatar/examples/sizes.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Avatar from '../../../../../Avatar';
+import IconCloud from 'mineral-ui-icons/IconCloud';
+import DemoLayout from '../components/DemoLayout';
+
+export default {
+  id: 'sizes',
+  title: 'Sizes',
+  description: `Avatar is available in the following sizes.`,
+  scope: { Avatar, DemoLayout, IconCloud },
+  source: `
+    <DemoLayout>
+      <Avatar size="small">
+        <img src="/images/Avatar.svg" alt="Sam" />
+      </Avatar>
+      <Avatar size="small">Sam</Avatar>
+      <Avatar size="small">
+        <IconCloud title="cloud" />
+      </Avatar>
+      <br />
```

Comment:
Suggest to add line breaks following the `br` tags for readability.

Reply:
Done.